lib/arturo/feature_caching.rb

```diff
@@ -60,9 +61,8 @@ def fetch(cache, symbol, &block)
           features = cache.read("arturo.all")
 
           unless cache_is_current?(cache, features)
-            features = Hash[Arturo::Feature.all.map { |f| [f.symbol.to_sym, f] }]
-            mark_as_current!(cache)
-            cache.write("arturo.all", features, :expires_in => Arturo::Feature.cache_ttl * 10)
+            features = arturos_from_origin(features)
```

Comment:
can we rename both of these variables? Maybe the method argument could be `existing_features` and the new variable could be `updated_features`? reusing the name `features` everywhere makes the code a little harder to follow.